llvm/test/Other/dump-before-after.ll

```diff
@@ -0,0 +1,57 @@
+; RUN: mkdir -p %t/logs
```

Comment:
Is the `mkdir` needed?
On Linux, `rm -rf` ignores missing directories, not sure about Windows.